modules/liftover/1.0/config/default.yaml

```diff
@@ -0,0 +1,29 @@
+lcr-modules:
+
+    liftover:
+        inputs:
+            sample_seg: "data/{sample_id}_subclones.hg38.igv.seg"
+        scratch_subdirectories: "scratch/"
+        options:
+            seghg38tobedhg38: "{MODSDIR}/src/convert_for_liftover.py"
+            bedhg38tobedhg19: "{MODSDIR}/src/chains/hg38ToHg19.over.chain"
```

Comment:
Did you mean the global reference directory that is created by the utils module before running star? Or it will be better to create the "{MODSDIR}/reference"?

Reply:
Yes, in essence, the references workflow would eventually need to be updated to pull these files down from UCSC for each reference build.

Reply:
If the file is a modest size I'm fine with it being here instead

Reply:
The size of the chain file is 3.9Mb for one reference build. I think that it can be here for now, but with the next update of reference files downloading liftover chains can be included, and then I will update the link to the location of reference chain in this config